Magento_Customer/templates/address/edit.phtml

```diff
@@ -19,92 +19,92 @@
                 <?= /* @escapeNotVerified */ $block->getNameBlockHtml() ?>
 
                 <div class="input dashboard__input">
-                    <?php 
+                    <?php
                         $getCompanyValidationClass = $this->helper('Magento\Customer\Helper\Address')
```

Comment:
since `$this->helper('Magento\Customer\Helper\Address')` used multiple time maybe it's better to declare new var to it

Reply:
right